File: Server/API/Models/Asset.cs
using System;
using System.Collections.Generic;

namespace Asset_API.Models
{
    public partial class Asset
    {
        public string PartId { get; set; }
        public string Description { get; set; }
        public string UseFor { get; set; }
        public string StorageBin { get; set; }
        public double? StockAmount { get; set; }
        public string Search { get; set; }
        public string Brand { get; set; }
        public bool? Local { get; set; }
        public bool? Import { get; set; }
        public string Note { get; set; }
        public string AlpSop { get; set; }
        public string Category { get; set; }
        public string Supplier { get; set; }
    }
}

File: Mobile App/AndroidAppScan/app/src/main/java/com/tool/scanqrasset/model/Asset.kt
package com.tool.scanqrasset.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Asset(
    @SerializedName("partId") val partId: String,
    @SerializedName("description") val description: String,
    @SerializedName("useFor") val useFor: String,
    @SerializedName("storageBin") val storageBin: String,
    @SerializedName("stockAmount") val stockAmount: Int,
    @SerializedName("search") val search: String,
    @SerializedName("brand") val brand: String,
    @SerializedName("local") val local: Boolean,
    @SerializedName("import") val import: Boolean,
    @SerializedName("note") val note: String,
    @SerializedName("alpSop") val alp: String,
    @SerializedName("note2") val note2: String,
    @SerializedName("supplier") val supplier: String,
) : Serializable